src/platform/site-wide/sass/modules/_m-campaign-landing-page.scss

```diff
@@ -1,36 +1,21 @@
-.va-c-story-teaser {
-  img {
-    max-height: 150px;
-  }
-}
-
-.va-c-white-box {
-  width: 220px;
-}
-
 .va-c-blue-line {
   width: 40px;
 }
 
-.va-c-li-box {
-  min-width: 280px;
-  width: 280px;
-}
-
-.va-c-benefit-category {
-  min-width: 280px;
-}
-
-.va-c-custom-hero-button {
+.va-u-box-shadow--none {
   box-shadow: none;
 }
 
-.va-c-hero {
+.va-u-background--image {
   background-position: center;
   background-repeat: no-repeat;
   background-size: cover;
 }
 
-.va-c-hero-content {
-  max-width: 400px;
+.va-u-background--gradiant-blue {
+  @include linear-gradient-background($color-old-browser-background-start, $color-old-browser-background-end);
+}
+
+.va-u-text-transform--uppercase {
+  text-transform: uppercase;
```

Comment:
Sooo since we ended up hardly needing any custom style it felt like using our own utility classes was somehow easier to deal with than tiny component classes